js/src/components/account-card/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Flex, FlexItem, FlexBlock } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import Subsection from '.~/wcdl/subsection';
+import { ReactComponent as GoogleLogo } from './gogole-g-logo.svg';
+import './index.scss';
+
+export const GOOGLE = 'google';
+
+const appearanceDict = {
+	[ GOOGLE ]: {
+		Logo: GoogleLogo,
+		title: __( 'Google account', 'google-listings-and-ads' ),
+	},
+};
+
+/**
+ * Renders a Card component with account info and status.
+ *
+ * @param {Object} props React props.
+ * @param {string} props.appearance Predefined account enum to indicate the card appearance.
```

Comment:
:nail_care: 
You can define enumtype in JSDoc

```suggestion
/**
 * Enum of account card appearances.
 *
 * @enum {{Logo: JSX.Element, title: string}}
 */
const appearanceDict = {
	[ GOOGLE ]: {
		Logo: GoogleLogo,
		title: __( 'Google account', 'google-listings-and-ads' ),
	},
};

/**
 * Renders a Card component with account info and status.
 *
 * @param {Object} props React props.
 * @param {appearanceDict} props.appearance Kind of account to indicate the card appearance.
```

Reply:
The `appearance` prop is string type, but the suggested change seems doesn't fit with it. I adjusted the doc by https://github.com/woocommerce/google-listings-and-ads/pull/838/commits/a52920e91605c8a926daabf50dbd1a7f2f6a12c1.